packages/lbfgs/lbfgs.0.9.5/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+maintainer: "Christophe Troestler <Christophe.Troestler@umons.ac.be>"
+authors: [ "Christophe Troestler <Christophe.Troestler@umons.ac.be>" ]
+license: "LGPL-3.0 with OCaml linking exception"
```

Comment:
it seems that the previous license is `LGPL-3.0-only WITH OCaml-LGPL-linking-exception` (which is slightly different than the one in this PR.) Is this change intentional?